framework/src/modules/nft/events/all_nfts_support_removed.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { BaseEvent, EventQueuer } from '../../base_event';
+
+export class AllNFTsSupportRemovedEvent extends BaseEvent<{}> {
+	public log(ctx: EventQueuer): void {
+		this.add(ctx, {});
```

Comment:
```suggestion
export class AllNFTsSupportRemovedEvent extends BaseEvent<undefined> {
	public log(ctx: EventQueuer): void {
		this.add(ctx, undefined);
```
probably better to use `undefined` to remove eslint complain. same for other events where applicable.